src/conda_devenv/devenv.py

```diff
@@ -428,62 +433,53 @@ def __call_conda_env_update(
     output_filename: Path,
     env_manager: Literal["conda", "mamba"],
 ) -> str | int | None:
-    command = [
-        env_manager,
+    cmdline = [
         "env",
         "update",
         "--file",
         str(output_filename),
     ]
     if not args.no_prune:
-        command.append("--prune")
+        cmdline.append("--prune")
     if args.name:
-        command.extend(["--name", args.name])
+        cmdline.extend(["--name", args.name])
     if args.quiet:
-        command.extend(["--quiet"])
+        cmdline.extend(["--quiet"])
     if args.verbose:
-        command.extend(["-" + "v" * args.verbose])
+        cmdline.extend(["-" + "v" * args.verbose])
 
     if not args.quiet:
-        print("> Executing: %s" % " ".join(command))
+        full_cmdline = [str(env_manager)] + cmdline
+        print(f"{Fore.CYAN}{' '.join(full_cmdline)}{Fore.RESET}")
 
-    old_argv = sys.argv[:]
-    try:
-        sys.argv = command if env_manager == "mamba" else command[1:]
-        try:
-            return _call_conda(env_manager)
-        except SystemExit as e:
-            return e.code
-    finally:
-        sys.argv = old_argv
+    return _call_conda(env_manager, cmdline)
 
 
-def _call_conda(env_manager: Literal["conda", "mamba"] = "conda") -> int:
+def _call_conda(env_manager: Literal["conda", "mamba"], args: Sequence[str]) -> int:
     """
     Calls conda-env or mamba directly using its internal API. ``sys.argv``
```

Comment:
It looks like the documentation needs an update now.

Reply:
Thanks!